app/assets/javascripts/application.js

```diff
@@ -3,11 +3,11 @@
 
 //= require jquery
 //= require jquery_ujs
-//= require jquery-ui/datepicker
-//= require jquery-ui/draggable
-//= require jquery-ui/droppable
-//= require jquery-ui/tabs
-//= require jquery-ui/dialog
+//= require jquery-ui/widgets/datepicker
+//= require jquery-ui/widgets/draggable
+//= require jquery-ui/widgets/droppable
+//= require jquery-ui/widgets/tabs
+//= require jquery-ui/widgets/dialog
```

Comment:
I think it might also be worth seeing if we actually use the dialog widget, and if we can get rid of it. I think we end up using bootstap's modal instead.